### DISCUSSION TITLE
sudo less installation

### DISCUSSION THREAD

**Discussion by jkroepke** [Homebrew/discussions#4212] [Getting started] [answered]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

I would like to install home brew on my linux system.

I read https://docs.brew.sh/Homebrew-on-Linux and I would like to quote it:

> The installation script installs Homebrew to /home/linuxbrew/.linuxbrew using sudo if possible and within your home directory at ~/.linuxbrew otherwise. Homebrew does not use sudo after installation.

Today, I tried the installation on unpriviliged user, but I was not able to successfully install homebrew on linux

```
joe@8239a70d4e4e:~$ /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
==> Checking for `sudo` access (which may request your password)...
Sorry, user joe may not run sudo on 8239a70d4e4e.
Insufficient permissions to install Homebrew to \"/home/linuxbrew/.linuxbrew\" (the default prefix).

Alternative (unsupported) installation methods are available at:
https://docs.brew.sh/Installation#alternative-installs

Please note this will require most formula to build from source, a buggy, slow and energy-inefficient experience.
We will close any issues without response for these unsupported configurations.
```

I also check https://docs.brew.sh/Installation#alternative-installs, but I'm not interest into alternative installation methods. Or does homebrew on my personal home directory (~/.linuxbrew) any downsides?

**Comment by jkroepke** [comment, marked as answer]
Looking at https://github.com/Homebrew/install/pull/702, it seems like the docs are outdated.

**Comment by gromgit** [comment]
Thanks for the catch, update inbound: https://github.com/Homebrew/brew/pull/14633.

> I also check https://docs.brew.sh/Installation#alternative-installs, but I'm not interest into alternative installation methods. Or does homebrew on my personal home directory (~/.linuxbrew) any downsides?

Everything else in https://docs.brew.sh/Homebrew-on-Linux still stands, only the reference to support for home dir installs will be removed. If you can't/won't install under the official directory prefix, then you'll have to use an alternative install method, likely https://docs.brew.sh/Installation#untar-anywhere-unsupported.
